application/controllers/admin/surveyadmin.php

```diff
@@ -427,6 +427,31 @@ public function activate($iSurveyID)
             $aData['failedgroupcheck'] = $failedgroupcheck;
             $aData['aSurveysettings'] = getSurveyInfo($iSurveyID);
 
+            if (empty(Yii::app()->request->getPost('activate'))){
+                $survey = Survey::model()->findByAttributes(array('sid' => $iSurveyID));
+
+                $todayDate = new DateTime('now');
+                $surveyExpireDate = new DateTime($survey->expires);
+                $surveyStartDate = new DateTime($survey->startdate);                
+
+                if (!empty($survey->startdate) && $todayDate < $surveyStartDate) {
+                    
+                    $aViewUrls['output'] = "<div class='messagebox ui-corner-all'><div class='warningheader'>".gT("This Survey is not yet started. Would like to change the survey settings?")."</div>";                        
```

Comment:
Because : show at same place where we see "ANonymous : YES/NO" etc ...
After you can add questoion etc ...

Seems better here : show the alert BEFORE activate :)